### PR TITLE
Add Nginx Proxy issue to docs

### PR DESCRIPTION
**Pull request by Asion001**
### Category
Documentation

### CODE REVIEW

File: docs/community/known-issues.md [hunk 1]
```diff
@@ -29,3 +29,13 @@ If you are using [custom icons](/docs/customizations/icons), you might be unable
 
 ### Resolution
 For the time being, restart your docker container after adding / making any changes to your icons.
+
+## 🚨 403 status on most of applications
```
Comment (manuel-rw) [inline docs/community/known-issues.md:33]:
```suggestion
## 🚨 Nginx Reverse Proxy may block Homarr's requests to several apps
```

File: docs/community/known-issues.md [hunk 2]
```diff
@@ -29,3 +29,13 @@ If you are using [custom icons](/docs/customizations/icons), you might be unable
...
+If you are using [Nginx Proxy Manager](https://nginxproxymanager.com) for opening Homarr.
```
Comment (manuel-rw) [inline docs/community/known-issues.md:34]:
```suggestion
If you are using [Nginx Proxy Manager](https://nginxproxymanager.com), some features like Torrent download clients or pings, may not work.
```

**Review by manuel-rw — CHANGES_REQUESTED** [review]

File: docs/community/known-issues.md (hunk 2 above)
Comment (Asion001) [reply docs/community/known-issues.md:34]: I think need to be more specific about opening Homarr instance through proxy.
Its ok if we write like this?

```suggestion
If you are using [Nginx Proxy Manager](https://nginxproxymanager.com) to open Homarr instance, some features like Torrent download clients or pings, may not work.
```

**Review by manuel-rw — APPROVED** [review]

**Review by ajnart — APPROVED** [review]

**Comment by ajnart** [comment]
Thank you @Asion001 !

**Pull request merged**